icu4c/source/tools/toolutil/pkg_genc.cpp

```diff
@@ -228,6 +228,17 @@ static const struct AssemblyType {
     },
 /* align 16 bytes */
 /*  http://msdn.microsoft.com/en-us/library/dwa9fwef.aspx */
+#if defined(_WIN64)
```

Comment:
I propose that we don't use `#if` but instead give this a different name like "masm64". (Or "masm", and rename the old one to "masm32".) That would let us generate either one on any machine.